.github/README.md

```diff
@@ -26,7 +26,7 @@ At Q-CTRL, we use a suite of microservices to support our products. Typically, t
 
 ### Requirements
 
-1. Create a new GitHub repository under your account which will contain your solution. This new repository should **not** be a fork of this repository and should **not** make reference to "Q-CTRL" in the repository name, description, or code.
+1. Create a new GitHub repository under your account which will contain your solution. The new repository which you are going to create, should **not** be a fork of this repository and should **not** make any reference to "Q-CTRL" in the repository name, description, or code.
```

Comment:
Nice suggestions, just one more minor change to make it even clear what we are requesting.

```suggestion
1. Create a new GitHub repository under your account which will contain your solution. The new repository which you are going to create, **must not** be a fork of this repository and **must not** make any reference to "Q-CTRL" in the repository name, description, or code.
```